pom.xml

```diff
@@ -67,6 +67,16 @@
             <version>1.18.30</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Please add a comment about the purpose of these dependencies as we do for the rest.